src/Liquid/Context.php

```diff
@@ -44,6 +44,13 @@ class Context
 	 */
 	public $environments = array();
 
+	/**
+	 * Called "sometimes" while rendering. For example to abort the execution of a rendering.
+	 *
+	 * @var null|callable
+	 */
+	protected $tickFunction = null;
```

Comment:
```suggestion
	private $tickFunction = null;
```

Reply:
i am 👍 with this, the reason i used `protected` was to be consistent with `$filterbank` and `$assigns`